.github/workflows/test-pg_search.yml

```diff
@@ -227,7 +219,7 @@ jobs:
       - name: Test Packaging pg_search
         if: steps.check_skip.outputs.skip_remaining_steps != 'true' && github.base_ref == 'main'
         working-directory: pg_search/
-        run: cargo clean && cargo pgrx package --features icu
+        run: cargo pgrx package --features icu
```

Comment:
Q:  Why do we do this at all?  This seems like it's just testing pgrx... pgrx has its own CI that tests itself.

Reply:
The idea was to test that the extension packages successfully. We only do this on PRs to `main` so that when we run `cargo pgrx package` on the deployment, we know it'll succeed.

Reply:
There’s nothing in an an extension (pg_search or otherwise) that influences the decisions “cargo pgrx package” makes. 

I’m not sure this test tells us anything valuable. It just spins the CPUs for 5 or 6 minutes.

Reply:
Alright, well let's remove it then :)